//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// ac_O_P.h
//
// Code generation for function 'ac_O_P'
//

#ifndef AC_O_P_H
#define AC_O_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void ac_O_P(double P_init, double V_init, double A_init, double P_wayp,
            double V_max, double J_max, double J_min, creal_T t[21]);

#endif
// End of code generation (ac_O_P.h)
